pkg/ctrdtaskapi/update.go

```diff
@@ -15,3 +16,10 @@ type PolicyFragment struct {
 	// fragment and any additional information required for validation.
 	Fragment string `json:"fragment,omitempty"`
 }
+
+type ContainerMount struct {
+	HostPath      string
+	ContainerPath string
+	ReadOnly      bool
+	Type          string
```

Comment:
should we make this a specific type? what are the values we envision for this?

Reply:
> should we make this a specific type? what are the values we envision for this?

Oh like a const for all the types we support? in hcsshin we use plain strings for the types supported all over - example: https://github.com/search?q=repo%3Amicrosoft%2Fhcsshim%20%22virtual-disk%22&type=code to name a few types. Should I add the mount types and change the existing files in a separate PR and then update this PR to take in the changes?

Reply:
https://github.com/microsoft/hcsshim/pull/1920 submitted a separate PR to create a list of supported mounts on hcsshim today

Reply:
will those mount type constants be exposed here as well?

Reply:
yeah I'll change the type of this field once PR #1920 goes in.

Reply:
#1920  was checked in. Reflected the updated mount types in this PR.